doc/modules/clustering.rst

```diff
@@ -244,6 +244,9 @@ to the dataset :math:`X`.
 * :ref:`sphx_glr_auto_examples_cluster_plot_kmeans_plusplus.py`: Using K-means++
   to select seeds for other clustering algorithms.
 
+* :ref:`sphx_glr_auto_examples_cluster_plot_kmeans_stability_low_dim_dense.py`:
+  Evaluating the impact of k-means initializations to convergence.
```

Comment:
Rather than adding this here without context, I would prefer to add this above at "For a detailed example of comaparing different initialization schemes, refer to :ref:`sphx_glr_auto_examples_cluster_plot_kmeans_digits.py`." (Would be nice if you could also correct the spelling mistake in comaparing.)